net/bind/Makefile

```diff
@@ -56,7 +55,7 @@ endef
 define Package/bind-libs
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+libopenssl +zlib
+  DEPENDS:=+libopenssl +zlib +libpthread +(mips||mipsel||TARGET_gemini):libatomic
```

Comment:
I don't like this. How about we just add libatomic for all targets?

Reply:
What don't you like about it? We don't want unnecessary dependencies. If libatomic is only needed for certain targets, it should only be a dependency on those targets.

Reply:
Sure. The main concern is whether or not this is a moving target. libatomic itself seems to be around 6kb. Not too big.

Reply:
OK, I've looked through the rest of the packages repository, and haven't found any others that try to conditionally enable the libatomic dependency. They all add it globally if they use it at all. So let's just follow that precedent.

Reply:
I see